inference/core/workflows/core_steps/common/query_language/entities/enums.py

```diff
@@ -43,3 +43,8 @@ class ImageProperty(Enum):
     SIZE = "size"
     HEIGHT = "height"
     WIDTH = "width"
+
+
+class DetectionsSelectionMode(Enum):
+    LEFT_MOST = "left_most"
+    RIGHT_MOST = "right_most"
```

Comment:
top_most, bottom_most, center_most?

Would this also be where largest, smallest would go?

Reply:
I guess so - I am open to discuss UQL in general, as this is very initial draft, I do not have strong opinions yet on lots of topics related to this.